requirements.txt

```diff
@@ -1,13 +1,26 @@
+cached-property==1.4.3
+certifi==2018.4.16
+chardet==3.0.4
+coverage==4.5.1
+coveralls==1.3.0
+defusedxml==0.5.0
 Django==2.0.7
 django-cors-middleware==1.3.1
 django-environ==0.4.5
 django-extensions==2.1.0
+django-filter==2.0.0
 djangorestframework==3.8.2
+docopt==0.6.2
```

Comment:
Is this package necessary? Are you using it anywhere?

Reply:
I didn't use it. Let me remove it too.